CMakeLists.txt

```
@@ -3,7 +3,7 @@
 
 cmake_minimum_required(VERSION 3.16)
 
-project(iDynTree VERSION 11.0.0
+project(iDynTree VERSION 12.0.0
```

```suggestion
project(iDynTree VERSION 11.1.0
```

The change done in the PR are ABI backward compatible, no need to bump the major version.